deploy/lib/setIamPolicy.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const _ = require('lodash');
+const BbPromise = require('bluebird');
+
+module.exports = {
+  setIamPolicy() {
+    return BbPromise.bind(this).then(this.getFunctions).then(this.setPolicies);
+  },
+
+  getFunctions() {
+    const project = this.serverless.service.provider.project;
+    const region = this.options.region;
+
+    const params = {
+      parent: `projects/${project}/locations/${region}`,
+    };
+
+    return this.provider
+      .request('cloudfunctions', 'projects', 'locations', 'functions', 'list', params)
+      .then((response) => {
+        return response.functions;
+      });
+  },
+
+  setPolicies(functions) {
+    const policies = this.serverless.service.provider.functionIamBindings;
+
+    // If there are no IAM policies configured with any function, there is nothing to
+    // do here.
+    if (!policies || !Object.keys(policies).length) {
+      return BbPromise.resolve();
+    }
+    this.serverless.cli.log('Setting IAM policies...');
+
+    _.forEach(policies, (value, key) => {
```

Comment:
Let's use `Object.entries().forEach` instead (no point to use lodash, when we can do same with native functions)